website/dashboard/test_plots.js

```diff
@@ -0,0 +1,54 @@
+document.addEventListener('DOMContentLoaded', function() {
+    fetch('report-summary.json')
+        .then(response => response.json())
+        .then(data => {
+            function createPlot(container, parameter) {
+                var reference_values = data.map(d => d[parameter]);
+                var fit_values = data.map(d => d[parameter + '_fit']);
+                var errors = fit_values.map((d, i) => d - reference_values[i]);
+
+                var tolerance = reference_values.map((d, i) => data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d);
+                var negative_tolerance = reference_values.map((d, i) => -(data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d));
```

Comment:
Could simplify to negative tolerance rather than running the whole equation again.